README.md

```diff
@@ -260,14 +285,37 @@ const changes = joey.diff(source, target);
 }
 ```
 
-### diffFiles(sourcePath, targetPath)
+### async diffFiles(sourcePath, targetPath, optionalOutputPath)
 
-A helper method to work with files.
+```js
+const results = await joey.diffFiles(sourcePath, targetPath, optionalOutputPath);
+```
+
+`results` is an array of objects like:
 
 ```js
-const changes = await joey.diff(sourcePath, targetPath);
+[
+  {
+    source: 'path to the source file',
+    target: 'path to the target file',
+    changes: [
+      // see above
+    ],
+  },
+  {
+    // ...
+  },
+]
 ```
 
+You can diff:
+
+- one `source` file against many, if `target` is a directory
+- many source files against one `target`, if `source` is a directory
+- matching pairs of files if `source` and `target` are directories (the files with the same names in both `source` and `target` will be diffed)
+
+`optionalOutputPath` can be either a file or a directory. In case of a directory, for each file matched, a file with the same name will be created in it.
```

Comment:
What is the naming policy in the output for a folder when there are just one file as source or target? I suppose that the many files are used for it but it is not clear here.

Reply:
It will throw an error stating that it cannot be a folder.